client/src/App.js

```diff
@@ -5,10 +5,12 @@ import Login from "./components/Login/Login";
 import Meetup from "./components/Meetup/Meetup";
 import Profile from "./components/Profile/Profile";
 import Search from "./components/Search/Search";
+import Navbar from "./components/Navbar/Navbar";
 
 function App() {
   return (
     <Router>
+      <Navbar />
       <div>
```

Comment:
We can probably remove this `div` now that we have a nav bar

Reply:
I took out the div but left the Search link since the navbar doesn't have a searchbar. Speaking of - should we add a search bar to the navbar?

Reply:
Not sure if we want to put the search bar in the navbar, but we can discuss how we want to do it later.